app-web/__tests__/pages/resourceType.test.js

```diff
@@ -131,11 +132,36 @@ describe('Resource Type Template Page', () => {
     expect(Alert).toBeInTheDocument();
   });
 
+  test('Cards are not duplicating after multiple searches', () => {
+    queryString.parse.mockReturnValue({ q: 'foo' });
+    useSearch.mockReturnValue([{ id: SIPHON_NODES[0].id }]);
+    const { rerender, queryAllByText } = render(
+      <ThemeProvider theme={theme}>
+        <ResourceType {...props} />
+      </ThemeProvider>,
+    );
+
+    const startingNumCards = queryAllByText('Documentation').length;
```

Comment:
replace 'Documentation' with the `RESOURCE_TYPES.DOCUMENTATION` constant

Reply:
Done!